new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java

```diff
@@ -137,4 +232,162 @@ private static void writeToFile(Object obj, String resultDir, String fileName) t
     }
   }
 
-}
+  /**
+   * Copy files from persistent volume to local folder.
+   * @param namespace name of the namespace
+   * @param hostPath the persistent volume host path
```

Comment:
what does hostPath mean in this context?  there is a storage provider in k8s called HostPath -- are you talking about that? if not, pick a better/different name to avoid confusion